app/controllers/help_controller.rb

```diff
@@ -2,6 +2,8 @@ class HelpController < ApplicationController
   skip_before_action :authenticate_user!
 
   def new
+    return redirect_to signed_in_new_help_path if !!current_user
```

Comment:
I don't think you need the `!!` here, just `if current user` will do.

And as a matter of style, you could possibly rejig this to make it more readable, eg:
```
redirect_to signed_in_new_help_path && return if current_user
```
But that last point is just a matter of style.

Reply:
Tried that style suggestion but it didn't work. Thanks!

Reply:
Ah I see, this works, but the linter complains about the `and` (it prefers `&&`):
```
redirect_to signed_in_new_help_path and return if current_user
```

I still think we can improve the readability of this. It strikes me as quite hard to understand if you are new to this file. How about a before_action such as:
```
  before_action :redirect_signed_in_user, only: :new


private

  def redirect_signed_in_user
    return redirect_to signed_in_new_help_path if current_user
  end
```
This way it doesn't pollute the `new` action which makes it more readable, the method itself explains the action much better and it actually gets called before the new action, which is kind of what we want. Even though you should try not to overdo `before_action`s, this is the classic example of a before_action use.